add Refreshing Breeze

ntseng opened the pull request:
Summary
-------
- add gear Refreshing Breeze (cures a random debuff for each ally)

Local Tests Performed
---------------------
- [x] bot still turns on (no BuildErrors or circular dependencies)
- [x] cleanses on each party member
- [x] cleanses 2 debuffs on crit if target has 2 or more debuffs
- [x] cleanses 1 debuff if target has exactly 1 debuff
- [x] doesn't crash if target has 0 debuffs
- [x] cleanses 1 debuff on crit if target has 2 or more debuffs and 1 stack of Stasis
- [x] doesn't crash if target has debuffs and 2 or more stacks of Stasis

Issue
-----
N/A

ntseng committed:
add Refreshing Breeze

ntseng committed:
add needsLivingTargets to Poison Torrent and Refreshing Breeze

doubledh commented on source/gear/refreshingbreeze-base.js:
```diff
@@ -0,0 +1,48 @@
+			if (removedDebuffs.length > 1) {
+				resultTexts.push(`${target.getName(adventure.room.enemyIdMap)} is cured of ${removedDebuffs[0]} and ${removedDebuffs[1]}.`)
+			} else if (removedDebuffs.length > 0) {
+				resultTexts.push(`${target.getName(adventure.room.enemyIdMap)} is cured of ${removedDebuffs[0]}.`)
```
is there a text for if an adventurer whiffs on cleaning any debuffs?

doubledh reviewed (approved):
one ask about whiff message

ntseng committed:
add whiff text

ntseng committed:
Merge branch 'main' into refreshing-breeze

ntseng committed:
Merge branch 'v0.13.0' into refreshing-breeze

ntseng committed:
Merge branch 'main' into refreshing-breeze

ntseng committed:
Merge branch 'v0.13.0' into refreshing-breeze

ntseng committed:
fix refreshing breeze adding stagger to allies

ntseng committed:
add Shattering Sabotage Kit

ntseng committed:
Merge branch 'v0.13.0' into refreshing-breeze

ntseng committed:
Merge branch 'v0.13.0' into refreshing-breeze

ntseng committed:
add Shattering Corrosion
- fix price on Shattering Sabotage Kit
- fix missing entries in Debug Dungeon
- fix Frail damage being 25 per stack instead of 10

ntseng committed:
update changelog

ntseng committed:
Spiral Funnel works on Frail
- interface improvements for dealModifierDamage
- increase Frail damage to 20 per stack
- improve/fix Frail description

ntseng committed:
add more discord constants

ntseng committed:
rn fixes
- fix whiff when straddling the wrap boundary
- fix rn progression not wrapping in viewstartingartifact

ntseng committed:
avoid locking players out of picking moves when stunned

ntseng committed:
fix crash in enemy move generation

ntseng committed:
fix bad type in generateRandomNumber argument
